testing/ceph/APKBUILD

```diff
@@ -0,0 +1,554 @@
+	mv "$pkgdir"$_libexecdir/ceph/ceph-osd-prestart.sh "$subpkgdir"$_libexecdir/ceph
+
+	mkdir -p "$subpkgdir"$_udevrulesdir
+	mv "$pkgdir"$_udevrulesdir/60-ceph-by-parttypeuuid.rules "$subpkgdir"$_udevrulesdir
+	mv "$pkgdir"$_udevrulesdir/95-ceph-osd.rules "$subpkgdir"$_udevrulesdir
+
+	install -m 750 -o ceph -g ceph -d "$subpkgdir"$_localstatedir/lib/ceph/osd
+}
+
+librados() {
+	pkgdesc="RADOS distributed object store client library"
+	mkdir -p "$subpkgdir"/usr/lib
+	mv "$pkgdir"/usr/lib/librados.so.* "$subpkgdir"/usr/lib
+}
+
+librados_dev() {
+	pkgdesc="RADOS distributed object store client library headers"
+	depends="librados"
+	mkdir -p "$subpkgdir"$_includedir/rados
+	mv "$pkgdir"$_includedir/rados/librados.h "$subpkgdir"$_includedir
+	mv "$pkgdir"$_includedir/rados/librados.hpp "$subpkgdir"$_includedir
+	mv "$pkgdir"$_includedir/rados/buffer.h "$subpkgdir"$_includedir
+	mv "$pkgdir"$_includedir/rados/buffer_fwd.h "$subpkgdir"$_includedir
+	mv "$pkgdir"$_includedir/rados/page.h "$subpkgdir"$_includedir
+	mv "$pkgdir"$_includedir/rados/crc32c.h "$subpkgdir"$_includedir
+	mv "$pkgdir"$_includedir/rados/rados_types.h "$subpkgdir"$_includedir
+	mv "$pkgdir"$_includedir/rados/rados_types.hpp "$subpkgdir"$_includedir
+	mv "$pkgdir"$_includedir/rados/memory.h "$subpkgdir"$_includedir
+	mv "$pkgdir"$_libdir/librados.so "$subpkgdir"$_libdir
+	mv "$pkgdir"$_bindir/librados-config "$subpkgdir"$_bindir
+}
+
+librgw() {
+	pkgdesc="RADOS gateway client library"
+	depends="librados"
+	mkdir -p "$subpkgdir"$_libdir
+	mv "$pkgdir"$_libdir/librgw.so.* "$subpkgdir"$_libdir
+}
+
+librgw_dev() {
+	pkgdesc="RADOS gateway client library headers"
+	depends="librados"
+
+	mkdir -p "$subpkgdir"$_includedir/rados
+	mv "$pkgdir"$_includedir/rados/librgw.h "$subpkgdir"$_includedir/rados
+	mv "$pkgdir"$_includedir/rados/rgw_file.h "$subpkgdir"$_includedir/rados
+	mv "$pkgdir"$_libdir/librgw.so "$subpkgdir"$_libdir
+}
+
+py_rados() {
+	pkgdesc="Python libraries for the RADOS object store"
+	depends="librados"
+	mkdir -p "$subpkgdir"$_python_sitelib
+	mv "$pkgdir"$_python_sitelib/rados.so "$subpkgdir"$_python_sitelib
+	mv "$pkgdir"$_python_sitelib/rados-*.egg-info "$subpkgdir"$_python_sitelib
+}
+
+libradosstriper() {
+	pkgdesc="RADOS striping library"
+	depends="librados"
+	mkdir -p "$subpkgdir"$_libdir
+	mv "$pkgdir"$_libdir/libradosstriper.so.* "$subpkgdir"$_libdir
+}
+
+libradosstriper_dev() {
+	pkgdesc="RADOS striping library headers"
+	depends="libradosstriper librados-dev"
+	mkdir -p "$subpkgdir"$_includedir/radosstriper
+	mv "$pkgdir"$_includedir/radosstriper "$subpkgdir"$_includedir/radosstriper
+	mv "$pkgdir"$_libdir/libradosstriper.so "$subpkgdir"$_libdir
+}
+
+librbd() {
+	pkgdesc="RADOS block device client library"
+	depends="librados"
+	mkdir -p "$subpkgdir"/usr/lib
+	mv "$pkgdir"/usr/lib/librbd.so.* "$subpkgdir"/usr/lib
+
+	# TODO: look into this
+	#mkdir -p /usr/lib64/qemu/
+	#ln -sf %{_libdir}/librbd.so.1 /usr/lib64/qemu/librbd.so.1
```

Comment:
this looks like copied from fedora? not sure we need that as we do not Multilib.